doc/rst/developer/bestPractices/Sanitizers.rst

```diff
@@ -34,6 +34,15 @@ To only sanitize executables and not the compiler:
      unset CHPL_SANITIZE
      export CHPL_SANITIZE_EXE=address
 
+To get stack traces for the generated executable in terms of Chapel
+source line numbers, try compiling with:
+
+     chpl --cpp-lines -g --no-inline --savec <dirname> <program.chpl>
```

Comment:
This renders a little odd in rst -- https://github.com/chapel-lang/chapel/blob/acabcdb5129b08bd4d332d463df1f2b10874ebbf/doc/rst/developer/bestPractices/Sanitizers.rst

I'd probably use `.. code-block:: bash` for the compile lines

Reply:
How much does `--no-inline` help readability of the trace? It seems like that would really depend on what you're trying to debug (something inside a function or the callsite), but it has the potential to really hurt performance since we won't inline primitive things like `+` and others

Reply:
I always throw `--no-inline` when debugging and haven't noticed significant noise w.r.t. it.  Inlined `+` is almost always at the bottom of the callstack, so it's rare that inlined `+` causes extra noise.  Yet larger things that are inlined can be confusing when you don't get a stack frame corresponding to them (I/O comes to mind).